supplier/tests/test_supplier.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from conftest import Fields, TestFields
+from django.core.exceptions import FieldDoesNotExist, ValidationError
+from supplier.models import Supplier
+
+
+class TestSupplierModel:
+    @pytest.mark.django_db()
+    def test_save_supplier(self, supplier0):
+        assert supplier0 not in Supplier.objects.all()
+        Supplier.save_supplier(supplier0)
+        assert supplier0 in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, saved_supplier0):
+        assert saved_supplier0 in Supplier.objects.all()
+        Supplier.delete_supplier(saved_supplier0)
+        assert saved_supplier0 not in Supplier.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_supplier(self, supplier0):
```

Comment:
you have the same test twice name at line 15  + 21

Reply:
good call, fixed it👍